luacom/tLuaCOM.cpp

```
@@ -422,58 +528,61 @@ DWORD tLuaCOM::addConnection(tLuaCOM *server)
   {
-    connection_point->Release();
-    connection_point = NULL;
-
+    if(SUCCEEDED(hr))
+      connection_point->Unadvise(connection_point_cookie);
     return 0;
   }
 
   if(conn_point!=NULL)
+  {
+    HRESULT old_connection_hr = conn_point->Unadvise(conn_cookie);
```

Clang (14.0.6)

[warning] variable 'old_connection_hr' is not initialized

(cppcoreguidelines-init-variables)